lib/generators/react_on_rails/base_generator.rb

```diff
@@ -73,12 +73,12 @@ def update_application_js
           //= require bootstrap-sprockets
         DATA
 
-        application_js_path = "app/assets/javascripts/application.js"
-        application_js = dest_file_exists?(application_js_path) || dest_file_exists?(application_js_path + ".coffee")
-        if application_js
-          prepend_to_file(application_js, data)
+        app_js_path = "app/assets/javascripts/application.js"
+        found_app_js = dest_file_exists?(app_js_path) || dest_file_exists?(app_js_path + ".coffee")
+        if found_app_js
+          prepend_to_file(found_app_js, data)
         else
-          puts_setup_file_error("#{application_js} or #{application_js}.coffee", data)
+          create_file(app_js_path, data << "\n\n//= require turbolinks\n\n")
```

Comment:
Why add turbolinks? It's not a requirement of react_on_rails. We support it, but don't necessarily encourage it.

I'd say remove this.